docs/zkapps/snarkyjs/index.md

```diff
@@ -12,12 +13,14 @@ zkApps can now be deployed to Berkeley Testnet.
 
 # SnarkyJS
 
-SnarkyJS is a TypeScript library for
+SnarkyJS is a general-purpose zk framework that gives you the tools to create zk proofs. It lets you write arbitrary zk programs leveraging a rich set of built-in provable operations, like basic arithmetic, hashing, signatures, boolean operations, comparisons, and more. Use the SnarkyJS framework to write zkApps on Mina, smart contracts that execute client-side and have private inputs.
 
-- writing general-purpose zk programs
-- writing zk smart contracts for Mina
+SnarkyJS is a TypeScript (TS) library for:
 
-This is code that you might write when using SnarkyJS:
+- Writing general-purpose zk programs
+- Writing zk smart contracts for Mina
```

Comment:
now that you moved the paragraph about what snarkyjs is to the top, we kind of say the same thing twice. the bullet points enumerate the same thing we said in the paragraph before. Why do you think the paragraph should be moved to the top?

Reply:
I see your point and question... thinking about discoverability and how information is consumed, it seems like we "buried the lede" by placing the core description of utility (the "why") after the code sample

From my non-dev perspective, I did not evaluate the content as saying the same thing. I did like have the "why you need SnarkJS to write zkApp" statement in the first paragraph "Use the SnarkyJS framework to write zkApps on Mina, smart contracts that execute client-side and have private inputs."

wdyt? leave as is for now?